runtime/nls/j9cl/j9jcl.nls

```diff
@@ -580,3 +580,11 @@ J9NLS_JCL_CRIU_LOADING_OPTIONS_FILE_FAILED.explanation=CRIUSupport::checkpointJV
 J9NLS_JCL_CRIU_LOADING_OPTIONS_FILE_FAILED.system_action=The JVM will throw a JVMRestoreException.
 J9NLS_JCL_CRIU_LOADING_OPTIONS_FILE_FAILED.user_response=Check the arguments to CRIUSupport::registerRestoreOptionsFile.
 # END NON-TRANSLATABLE
+
+J9NLS_JCL_CRIU_RESTORE_INITIALIZE_TRACE_FAILED=The JVM could not initialize trace with the restore option file
+# START NON-TRANSLATABLE
+J9NLS_JCL_CRIU_RESTORE_INITIALIZE_TRACE_FAILED.sample_input_1=1
+J9NLS_JCL_CRIU_RESTORE_INITIALIZE_TRACE_FAILED.explanation=CRIUSupport::checkpointJVM failed.
+J9NLS_JCL_CRIU_RESTORE_INITIALIZE_TRACE_FAILED.system_action=The JVM will throw a JVMRestoreException.
+J9NLS_JCL_CRIU_RESTORE_INITIALIZE_TRACE_FAILED.user_response=Check the arguments to CRIUSupport::registerRestoreOptionsFile.
```

Comment:
Check documentation for CRIUSupport restore options

Reply:
Changed